src/main/scala/org/constellation/primitives/TransactionExt.scala

```diff
@@ -86,11 +86,6 @@ trait TransactionExt extends NodeData with Ledger with MetricsExt with PeerInfo
     tx
   }
 
-  def handleSendRequest(s: SendToAddress): StandardRoute = {
-    val tx = createTransaction(s.dst, s.amount, s.normalized)
-    complete(tx.prettyJson)
```

Comment:
`Accept: text/plain` is used for `complete(tx.prettyJson)`. Should be `complete(tx)` which uses `Accept: application/json`